app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
     <title><%= yield :title %> - GOV.UK</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="https://www.gov.uk/favicon.ico" type="image/x-icon" />
+    <meta name="robots" content="noindex">
```

Comment:
Why are we noindexing?

Reply:
All journeys to services should start on GOV.UK: https://www.gov.uk/service-manual/technology/get-a-domain-name#telling-search-engines-not-to-index-pages

Reply:
But you should add `nofollow` to this directive too.

Reply:
Thanks, added.